doxygen/src/pack_publish.txt

```diff
@@ -1,75 +1,225 @@
+  </releases>
+…	
+\endcode
+
+When updating a pack, you need to:
+- Change the \elem{url} to the location of the latest pack file in the PDSC.
+- Add a new \elem{release} version with the correct entries in the PDSC.
+- Attach the pack file to the release assets.
+- Update the PIDX file with the latest version information.
+
+\note
+While the repository's \c main branch can be used to host the PDSC file, additional confidence against unwanted changes can
+be made if you use a dedicated branch for hosting the PDSC file. This avoids breaking unintentionally the pack location when
+accepting a pull request (PR).
 
-\section cp_KeilComPack Publish with Pack Index Service
+\section cp_KeilComPack Using a pack index service
 
-Several companies maintain a Pack Index Server to make packs available within tools or web pages. Once you are ready to deploy packs via such services, you
-may send the URL of the \elem{vendor}.\elem{name}.pdsc or \elem{vendor}.pidx file to the following email addresses:
+Tool, software, or silicon vendors may provide a web infra-structure that lists packs that are available from multiple
+vendors. Companies like Arm or IAR provide such a service for making CMSIS-Packs available to a
+wider audience of developers. The packs are then found by pack management tools and their contents are listed on web sites,
+such as <a class="el" href="https://www.keil.arm.com/packs/" target="_blank">Arm's list of CMSIS-Packs</a>.
 
-- for Arm and Keil tools and web pages email to <a href="mailto:CMSIS@arm.com">CMSIS@arm.com</a>
-- for IAR tools email to <a href="mailto:CMSIS@arm.com">CMSIS@iar.com</a>
+Once you are ready to publish your packs, please send the URL of the \elem{vendor}.\elem{name}.pdsc or \elem{vendor}.pidx
+file to the following email addresses:
 
-The *.pidx file allows you to change pack versions and add packs to your distribution list. All packs that are referenced in the
-*.pidx file will be processed and validated. <b>Only packs that pass the
-<a href="https://github.com/Open-CMSIS-Pack/devtools/tree/main/tools/packchk">packchk</a> and schema check</b> will become
-available via web pages and development tools.
+- for Arm tools, email to <a href="mailto:CMSIS@arm.com">CMSIS@arm.com</a>.
+- for IAR tools, email to <a href="mailto:CMSIS@arm.com">CMSIS@iar.com</a>.
 
-For example, Arm uses the pack information to generate a list of available software packs. The following content will be processed in order to generate
-the web site
-(<a href="https://developer.arm.com/embedded/cmsis/cmsis-packs" target="_blank">https://developer.arm.com/embedded/cmsis/cmsis-packs</a>):
+Arm uses the pack information to generate a list of available software packs. The following content will be processed in
+order to generate the web site (<a href="https://www.keil.arm.com/packs/" target="_blank">https://www.keil.arm.com/packs/</a>):
 - Version
-- Release Notes
+- Release notes
```

Comment:
`<description>`  and `overview md file`.
Should we mention here that we need to register an email that the importer will send import issues to?